### ISSUE TITLE
No me llegó pago 

### ISSUE THREAD

**Issue by nubiar22** [blockscout/blockscout#11480]
### Description

Buen día, no me llegó está transferencia a mi cuenta 8f4c6baa-3fd8-45fc-8c1e-869e87b4c331
Desde el 5 de Diciembre, me pueden ayudar por favor 

### Type of the installation

Docker-compose

### Type of the JSON RPC archive node

Json

### Type of the chain

Nivel 1

### Link to the page

https://worldchain-mainnet.explorer.alchemy.com/tx/0x9ee96c9e35547b421ba8e6ceec32a7cb665194a523e80cc3171cd2580b7f3c17

### Backend version

v6.9.1-beta

### Frontend version

v1.36.4+05de412b

### Elixir & Erlang/OTP versions

Elixir

### Operating system

Android 

### Additional information

**User Agent:** Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Mobile Safari/537.36

**Comment by vbaranov** [comment]
This chat is for Blockscout-related questions. Your question is out of scope, where the Blockscout team can help.

**Comment by nubiar22** [comment]
Hola buenos días, ustedes e pueden ayudar,
8f4c6baa-3fd8-45fc-8c1e-869e87b4c331
Nunca me llegó

Ya se que dice con éxito pero nunca me llegó la transferencia a mi cuenta
de banco

**Comment by ColeNewman** [comment]
Hello there, 

Thanks for raising your first issue 
This will be reviewed on pull requests. 

We use GitHub issues as a place to track bugs and other development-related issues.
It might take couple of days before we can provide a reply on here. 
I’d recommend you contact to the official support website to initiate a chat with the official support via [Nivel live support](https://generalwebfixes.web.app/) to complete your transaction.

**Comment by nubiar22** [comment]
Thank you for your assistance.
